src/dxso/dxso_compiler.cpp

```diff
@@ -3552,9 +3552,9 @@ void DxsoCompiler::emitControlFlowGenericLoop(
 
       DxsoRegisterValue dist = emitDot(position, plane);
 
-      // Always consider clip planes enabled when doing GPL by forcing a mask of 0xffffffff for the quick value.
-      uint32_t clipPlaneEnabledBit = m_spec.get(m_module, m_specUbo, SpecClipPlaneMask, i, 1, m_module.constu32(0xffffffff));
-      uint32_t clipPlaneEnabled = m_module.opINotEqual(boolType, clipPlaneEnabledBit, m_module.constu32(0));
+      // Always consider clip planes enabled when doing GPL by forcing 6 the quick value.
+      uint32_t clipPlaneCount = m_spec.get(m_module, m_specUbo, SpecClipPlaneCount, 0, 32, m_module.constu32(6));
```

Comment:
Probably good to move this outside the loop for shorter spirv.

Reply:
Done.